scrapers/Pure-XXX.yml

```diff
@@ -0,0 +1,40 @@
+name: "Pure-XXX"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pure-xxx.com
+      - pure-ts.com
+      - pure-bbw.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="video_membership"]/div[@class="titlebox clear"]/h3/text()
+      Details: //div[@class="aboutvideo"]/p/text()
+      Performers:
+        Name: //div[@class="video_description"]/ul[@class="featuredModels"]/li/a/span/text()
+      Image:
+        selector: (//div[@id="logo"]//a/@href | //div[@class="flexslider"]//a/@href) | (//div[@class="videohere"]/img[contains(@class,"stdimage")]/@src | //script[contains(.,'jwplayer("jwbox").setup')]/text())
+        concat: '@~@'
+        postProcess:
+          - replace:
+              - regex: (.+@~@)(.+image:\s+")(.+jpg)(.+)
+                with: $1$3
+              - regex: \/\/(.*).com\/.*@~@(.*)
+                with: https://$1.com$2
+      Studio:
+        Name:
+          selector: //div[@id="logo"]//img/@alt | //div[@class="flexslider"]//a/@title
+          postProcess:
+            - replace:
+              - regex: (.*)\-(.*)
+              - with: $1 $2
```

Comment:
The validator doesn't like that you've got an extra dash in front of `with:` here, and this is also causing scrapes from pure-xxx.com to fail because the studio ends up empty

Reply:
Thank you for pointing it out. I have fixed the PR.